### DISCUSSION TITLE
Problem with remote access

### DISCUSSION THREAD

**Discussion by simjoc** [RaspAP/raspap-webgui#1203] [Q&A]
Hello,

I've a problem and hopefully somebody van help. I'm using the latest raspap insiders version.

I'm om holiday now and using the raspap to connect to 4g with an external antenna. This was working perfectly. Only the wifi signal from the raspberry didnt get good oudside the campervan. 

Now starts the stupid part. 
I've got a levelone usb adaptor (WUA--0614). The raspap recognise the device and in the hotspot settings I did change the default wlan to the levelone.

I can see my wifi network but I dont receive a ip adress. When I manually set my ip adress I can connect to the wifi. I then can't reach the web interface or connect thru putty. 

Now I didn't took my laptop or cardreader with me so I borrowed a laptop and utp cable. So the dhcp server doesnt work. When setting up a manual ip (10.3.141.2) I cant connect or ping the raspberry.

Does anybody have suggestion how to connect to the unit and put it batch to the default wlan adaptor?

Thanks!

**Comment by billz** [comment]
With a USB adaptor and borrowed laptop you can access your Pi using USB-OTG ("on the go") or gadget mode. This requires either a Pi Zero or a Pi4 Model B—the USB2 OTG port is integrated with the USB-C power for the latter. This also requires the ability to read/write to your SD card.

There are many tutorials online for how to enable this. Here's a popular one:
https://dev.webonomic.nl/4-ways-to-connect-your-raspberry-pi-4-to-the-internet

Once you have access, check the contents of `/etc/hostapd/hostapd.conf` and `/etc/dnsmasq.d/090_wlan0.conf`. Make any necessary changes to enable your other wlan adaptor, then restart RaspAP's systemd service to bring your AP back up:

```
sudo systemctl restart raspapd.service
```

Verify the status of your AP's services:

```
sudo systemctl status hostapd.service
sudo systemctl status dnsmasq.service
```

**Comment by simjoc** [comment]
Thank you for your fast reply! The OTG tric didn't work, I think my cable wasn't good enough. I did a reinstall and learned my lesson. 

Thank you for the quick support :)
